lib/colorprompt.g

```diff
@@ -13,6 +13,9 @@
 # see comment below
 if not IsBound(STDOut) then
   BindGlobal("STDOut", OutputTextUser());
+  if IsBound(HPCGAP) then
+    MakeReadOnly(STDOut);
+  fi;
```

Comment:
Actually, that was a hack I inserted in order to make the color prompt not trigger tons of errors. It works because `STDOut` essentially just contains the file descriptor 1, nothing else. 

I am not completely sure that it is entirely "correct", though. We may want to add a comment like `# FIXME: is this correct?`.

Well, or if you think it is correct, leave it as is; just wanted to make sure this is a conscious decision :-).